packages/core-mobile/ios/NotificationServiceExtensionInternal/NotificationService.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  NotificationService.swift
+//  NotificationServiceExtensionInternal
+//
+//  Created by Ruijia Lin on 9/11/24.
+//
+
+import UserNotifications
+import UIKit
```

Comment:
should we import UIKit here?

Reply:
[updated](238952ac669a47143fc5953850670c165d139c21)